docs/source/pywren.scripts.rst

```diff
@@ -0,0 +1,38 @@
+pywren\.scripts package
```

Comment:
If they are auto-generated do we need to commit these files ? Also does the script generate the `rst` files as well ?